src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
@KrzysztofCwalina, sorry for the noise. I saw some HTML class is going internal and there is a chance to extend a proposition; it reminded me of [zencoding](https://github.com/madskristensen/zencoding/) where consumers demand full-blown [Emmet features](http://docs.emmet.io/cheat-sheet/) which require hacking with such classes. I have no issue with this particular class, just sharing my 2 cents. :)